### PR TITLE
LPD-23506 Keeping characters in Forms success page message

### PR DESCRIPTION
**Pull request by rijema**
Hi team. 

The customer is facing some difficulties regarding the message description on Forms success page. 
My change is to remove the escape method when rendering the content. 

- The value was already treated before being saved in the database 
- The content is created by a person with permissions, not a random person that would be necessary to pay attention in every action as escaping the value defined by allowed users.
- Script tags and other actions are not being executed when rendering the page with the PR change.

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java [hunk 1]
```diff
@@ -548,10 +548,8 @@ public String getSuccessPageDescription(Locale locale)
 
 		LocalizedValue body = ddmFormSuccessPageSettings.getBody();
 
-		return HtmlUtil.escape(
-			GetterUtil.getString(
-				body.getString(locale),
-				body.getString(body.getDefaultLocale())));
+		return GetterUtil.getString(
+			body.getString(locale), body.getString(body.getDefaultLocale()));
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:552]: @rijema the `HtmlUtil` was added to fix [LPSA-56622](https://liferay.atlassian.net/browse/LPSA-56622) (see [commit](https://github.com/liferay-forms/liferay-portal/commit/52ac0e4f2471c0bf41ac494e5c8c4557bde46bae#diff-f5aa144644d14758bc53c5468b867b46ffc159b0bb95359cb2b26990a448bf4aR48)). 

Please check if you are not causing a regression by adding the `'">` value to the Success Page Description.

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java (hunk 1 above)
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:552]: Hi @carolmariaabb , thanks! Silly me. I concluded that if all the function was changed by Rodrigo, it wasn't a security ticket. I didn't get that far. [Screencast from 2024-04-17 07-48-31.webm](https://github.com/liferay-forms/liferay-portal/assets/41641596/d9ebc780-e23f-46cb-893d-f3b4e5da05d3)

Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:552]: @rijema so it's working fine?

Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:552]: Yes! I didn't receive any alert in the screen after submitting the forms and rendering the success page.

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by rijema** [comment]
Should I submit this PR manually, @carolmariaabb ?

**Comment by carolmariaabb** [comment]
@rijema you can send manually

**Pull request closed without merging**